### ISSUE TITLE
Status of this RFC

### ISSUE THREAD

**Issue by kulnor** [inadarei/rfc-healthcheck#78]
Greetings. I'm curious to know if this RFC is just dormant and if there is room to revive it. There is not much else out there...
Thanks!

**Comment by derekm** [comment]
Yes! People building Internet-based applications desperately need an RFC like this!

I tried to get people designing MicroProfile Health to come to this draft RFC when both projects were active.

Maybe this RFC got a little ambitious with reporting check details. Ridiculous opinions pervade this subject, which artificially limits collaboration.

a thorough analysis of popular health check APIs and their formats could help get people to set aside their closely held opinions
